recipes/onnxruntime/all/conanfile.py

```diff
@@ -27,11 +27,13 @@ class OnnxRuntimeConan(ConanFile):
         "shared": [True, False],
         "fPIC": [True, False],
         "with_xnnpack": [True, False],
+        "with_cuda_provider": [True, False],
```

Comment:
The `with_` prefix option implies that it will include another package. I recommend naming the option `use_cuda` as in the upstream, or alternatively `enable_cuda` , which I think conveys the intent more.

Reply:
Done!